x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/external/http/sender/RequestTask.java

```diff
@@ -87,7 +89,9 @@ private void notifyOfResult(Runnable runnable) {
             return;
         }
 
-        logger.debug(() -> format("Attempting to notify of result after already doing so for request [%s]", request.getRequestLine()));
+        logger.debug(
+            () -> format("Attempting to notify of result after already doing so for request from model id [%s]", request.modelId())
```

Comment:
I realize that this log message already had this wording, but I find it to be pretty confusing, do you think we should update it?

Specifically, I am not sure what "Attempting to notify" means. Also, what is the "result"? 

Generally, if I saw this log message, I wouldn't know why it was being logged, and I would have to read the code around it for context.

Reply:
> I realize that this log message already had this wording, but I find it to be pretty confusing, do you think we should update it?

Yeah that's fine. This message and various other parts of this class will actually be removed once batching is implemented.

I'll just remove it.